definitions/divorce/json/CaseEventToFields.json

```diff
@@ -7276,7 +7313,9 @@
     "PageLabel": "Grant Decree Absolute",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "DecreeAbsoluteGranted=\"nevershow\""
+    "FieldShowCondition": "DecreeAbsoluteGranted=\"nevershow\"",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
+    "RetriesTimeoutURLMidEvent": 120120
```

Comment:
"120,120"

Reply:
Existing files had 120120 and updated all refernces to "120,120" now